resources/nl-core/nl-core-Vaccination-event.xml

```diff
@@ -124,9 +124,14 @@
         <profile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-AnatomicalLocation" />
       </type>
       <binding>
-        <strength value="extensible" />
+        <strength value="required" />

```

Comment:
Restricting this from a use case is not in line with open world modelling. Could you slice this binding?